scipy/optimize/_linprog_ip.py

```diff
@@ -722,6 +722,14 @@ def _ip_hsd(A, b, c, c0, alpha0, beta, maxiter, disp, tol, sparse, lstsq,
 
     if disp:
         _display_iter(rho_p, rho_d, rho_g, "-", rho_mu, obj, header=True)
+    if callback is not None:
+        x_o, fun, slack, con, _, _ = _postsolve(x/tau, *_T_o,
```

Comment:
Here, would ``x/tau`` change ``x`` in-place? Can this leave interior-point to a similar error?

Reply:
No.

Reply:
Woops, I did now that I swear